src/App.tsx

```diff
@@ -583,9 +583,14 @@ const AppContent: React.FC<AppContentProps> = ({ project, setProject }): React.J
     if (project) {
       const tabs = createTabItemsFromProject(project);
       setTabItems(tabs);
-      setActiveTab(project.robot.modulePath);
+      
+      // Only set active tab to robot if no active tab is set or if the current active tab no longer exists
+      const currentActiveTabExists = tabs.some(tab => tab.key === activeTab);
+      if (!activeTab || !currentActiveTabExists) {
+        setActiveTab(project.robot.modulePath);
+      }
     }
-  }, [modulePathToContentText]);
+  }, [modulePathToContentText, activeTab]);
```

Comment:
I don't understand this change. According to the comment on line 581 "Update tab items when fetching modules is done", the purpose of this code is to update the tab items and to set the active tab to the robot.

I don't know why you've added activeTab to line 593.

Reply:
I can't figure out why it is there either.  My best bet is that I added it when I tried something more complicated and forgot to remove it.   I just removed it in another commit.